diff --git a/Dbus/interfaces/bash/BashInstall.cmake.in b/Dbus/interfaces/bash/BashInstall.cmake.in
deleted file mode 100644
index 49387cc..0000000
--- a/Dbus/interfaces/bash/BashInstall.cmake.in
+++ /dev/null
@@ -1,15 +0,0 @@
-IF("@ROOT_PREFIX@" STREQUAL "")
-	EXEC_PROGRAM("@PYTHON_EXECUTABLE@" "${CMAKE_CURRENT_BINARY_DIR}" ARGS
-					"setup.py" "install"
-					"--prefix=${CMAKE_INSTALL_PREFIX}"
-					"@DEBIAN_INSTALL_LAYOUT@")
-ELSEIF(NOT "@ROOT_PREFIX@" STREQUAL "" AND "@python_no_deb_layout@" STREQUAL "")
-	EXEC_PROGRAM("@PYTHON_EXECUTABLE@" "${CMAKE_CURRENT_BINARY_DIR}" ARGS
-					"setup.py" "install"
-					"--root=@ROOT_PREFIX@"
-					"@DEBIAN_INSTALL_LAYOUT@")
-ELSE()
-	EXEC_PROGRAM("@PYTHON_EXECUTABLE@" "${CMAKE_CURRENT_BINARY_DIR}" ARGS
-					"setup.py" "install"
-					"--root=@ROOT_PREFIX@")
-ENDIF()
diff --git a/Dbus/interfaces/bash/CMakeLists.txt b/Dbus/interfaces/bash/CMakeLists.txt
index fdc8d38..ae3cd42 100644
--- a/Dbus/interfaces/bash/CMakeLists.txt
+++ b/Dbus/interfaces/bash/CMakeLists.txt
@@ -1,16 +1,5 @@
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/CDBashApplet.py.in ${CMAKE_CURRENT_BINARY_DIR}/CDBashApplet.py)
 
-# the python script goes with other python scripts.
-if (NOT "${CMAKE_CURRENT_BINARY_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
-	# copy *.py to current binary dir (cleaner)
-	execute_process(COMMAND cp "${CMAKE_CURRENT_SOURCE_DIR}/setup.py" ${CMAKE_CURRENT_BINARY_DIR})
-	exec_program("${PYTHON_EXECUTABLE}" "${CMAKE_CURRENT_BINARY_DIR}" ARGS "setup.py" "-q" "build")
-endif()
-# Run a custom script with 'make install' 
-add_custom_target(bash_pyc ALL)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/BashInstall.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/BashInstall.cmake)
-set_target_properties(bash_pyc PROPERTIES POST_INSTALL_SCRIPT ${CMAKE_CURRENT_BINARY_DIR}/BashInstall.cmake)
-
 ########### install files ###############
 
 # and the bash functions goes into the data dir of the applet (didn't find a better place).
diff --git a/Dbus/interfaces/python/CMakeLists.txt b/Dbus/interfaces/python/CMakeLists.txt
deleted file mode 100644
index 988ed9d..0000000
--- a/Dbus/interfaces/python/CMakeLists.txt
+++ /dev/null
@@ -1,15 +0,0 @@
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/CDApplet.py.in ${CMAKE_CURRENT_BINARY_DIR}/CDApplet.py)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/CairoDock.py.in ${CMAKE_CURRENT_BINARY_DIR}/CairoDock.py)
-
-if (NOT "${CMAKE_CURRENT_BINARY_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
-	# copy *.py to current binary dir (cleaner)
-	execute_process(COMMAND cp "${CMAKE_CURRENT_SOURCE_DIR}/setup.py" ${CMAKE_CURRENT_BINARY_DIR})
-	exec_program("${PYTHON_EXECUTABLE}" "${CMAKE_CURRENT_BINARY_DIR}" ARGS "setup.py" "-q" "build")
-endif()
-
-########### install files ###############
-# Run a custom script with 'make install' 
-add_custom_target(python_pyc ALL)
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/PythonInstall.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/PythonInstall.cmake)
-set_target_properties(python_pyc PROPERTIES POST_INSTALL_SCRIPT ${CMAKE_CURRENT_BINARY_DIR}/PythonInstall.cmake)
-# execute_process(COMMAND python setup.py install ${DEBIAN_INSTALL_LAYOUT})
diff --git a/Dbus/interfaces/python/PythonInstall.cmake.in b/Dbus/interfaces/python/PythonInstall.cmake.in
deleted file mode 100644
index 49387cc..0000000
--- a/Dbus/interfaces/python/PythonInstall.cmake.in
+++ /dev/null
@@ -1,15 +0,0 @@
-IF("@ROOT_PREFIX@" STREQUAL "")
-	EXEC_PROGRAM("@PYTHON_EXECUTABLE@" "${CMAKE_CURRENT_BINARY_DIR}" ARGS
-					"setup.py" "install"
-					"--prefix=${CMAKE_INSTALL_PREFIX}"
-					"@DEBIAN_INSTALL_LAYOUT@")
-ELSEIF(NOT "@ROOT_PREFIX@" STREQUAL "" AND "@python_no_deb_layout@" STREQUAL "")
-	EXEC_PROGRAM("@PYTHON_EXECUTABLE@" "${CMAKE_CURRENT_BINARY_DIR}" ARGS
-					"setup.py" "install"
-					"--root=@ROOT_PREFIX@"
-					"@DEBIAN_INSTALL_LAYOUT@")
-ELSE()
-	EXEC_PROGRAM("@PYTHON_EXECUTABLE@" "${CMAKE_CURRENT_BINARY_DIR}" ARGS
-					"setup.py" "install"
-					"--root=@ROOT_PREFIX@")
-ENDIF()
